src/main/java/com/rakovets/course/java/core/practice/string/StringUtil.java

```diff
@@ -0,0 +1,302 @@
+package com.rakovets.course.java.core.practice.string;
+
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringUtil {
+
+    // task 1. Метод возвращает строку, которая является результатом склеивания двух строк.
+    // В случае строки равной null возвращает null.
```

Comment:
You couldn't duplicate task description.
```suggestion
    // task 1
```